backendImpl/src/test/java/com/virtuslab/gitmachete/backend/unit/GitMacheteRepositoryFactory_deriveParentAwareForkPointTestSuite.java

```diff
@@ -95,73 +94,4 @@ public void parentIsNotAncestorOfForkPointAndParentIsAncestorOfChild() {
     Assert.assertTrue(result.isDefined());
     Assert.assertEquals(parentCommit, result.get());
   }
-
-  @Test
```

Comment:
These tests are getting too complicated to maintain and bring relatively small value comparing to integration/functional ones